### DISCUSSION TITLE
Ability to set custom sort order for albums

### DISCUSSION THREAD

**Discussion by vishnukvmd** [ente-io/ente#879] [Enhancements]
Users might want to sort their albums in a specific order that's convenient to them. We can enable this by letting them long-press and drag and drop in the detailed album view.

https://github.com/ente-io/photos-app/blob/177b1bf3bb42589ccc0a73183a648b3f4023c8d6/lib/ui/collections/album/row_item.dart#L17

This could be a good use case for the [reorderables](https://pub.dev/packages/reorderables) package.

It'd be ideal to preserve this sort order on the server, but it's not a must have for the first cut.

**Comment by sneak** [comment]
It would also be useful if the albums could be sorted in reverse LRU (in the mobile album selection list, that is), so albums you have most recently modified are at the top.  This is a huge one for me as the album I almost always want to move something into is the album I last moved something into.  Having it in the same position at the top would be huge, as it's almost always offscreen and I have to search for it.

**Comment by swanitzek** [comment]
Are there any plans to implement this? Would be an amazing improvement!
